### PR TITLE
Fix broken find/replace across project

### PR DESCRIPTION
**Pull request by kristoferbaxter**
An accidental find/replace across the entire project seems to have caused a few mistakes in the LICENSE file.

Addresses https://github.com/ampproject/amphtml/issues/16272

### CODE REVIEW

File: examples/visual-tests/amp-by-example/LICENSE [hunk 1]
```diff
@@ -181,7 +178,7 @@
    APPENDIX: How to apply the Apache License to your work.
```
Comment (rsimha) [inline examples/visual-tests/amp-by-example/LICENSE]: Based on https://www.apache.org/licenses/LICENSE-2.0, I think we should end the license with `END OF TERMS AND CONDITIONS`.

File: examples/visual-tests/amp-by-example/LICENSE [hunk 2]
```diff
@@ -181,7 +178,7 @@
...
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
+      boilerplate notice, with the fields enclosed by brackets "[]"
```
Comment (rsimha) [inline examples/visual-tests/amp-by-example/LICENSE]: I think this section was (and still is) misleading. Per https://www.apache.org/licenses/LICENSE-2.0, the text that follows this sentence should read...
```
Copyright [yyyy] [name of copyright owner]
```
... and that isn't the case here. I think we should remove the appendix.

Comment (rsimha) [reply examples/visual-tests/amp-by-example/LICENSE]: Adding @cramforce for comment.

**Comment by nainar** [comment]
Pinging and assigning to @kristoferbaxter for next action.

File: examples/visual-tests/amp-by-example/LICENSE (hunk 2 above)
Comment (mrjoro) [reply examples/visual-tests/amp-by-example/LICENSE]: Perhaps we can address this in another PR and get this typo fix in?

**Comment by rsimha** [comment]
This is obsolete, since the file in question has been deleted.

**Pull request closed without merging**